src/rgw/rgw_rados.h

```diff
@@ -2595,10 +2582,6 @@ class RGWRados
 
   int create_pool(const rgw_pool& pool);
 
-  /**
-   * create a bucket with name bucket and the given list of attrs
-   * returns 0 on success, -ERR# otherwise.
-   */
   int init_bucket_index(RGWBucketInfo& bucket_info, int num_shards);
```

Comment:
Thanks @joscollin for reminding,  current prototype is tedious and obvious, can we just  drop the outdated doc?